examples/scripts/construct.yaml

```
@@ -5,6 +5,11 @@ channels:
   - http://repo.anaconda.com/pkgs/main/
 specs:
   - python
+
+extra_env_variables:
+  - CUSTOM_VARIABLE_1=CUSTOM1
+  - CUSTOM_VARIABLE_2=CUSTOM2
```

What about using the YAML dict syntax for this?

```suggestion
  - CUSTOM_VARIABLE_1: CUSTOM1
  - CUSTOM_VARIABLE_2: CUSTOM2
```

---

Sure! That helps validate thing at the yaml level so I am all for it.